### PR TITLE
Apply theme and acrylic changes without restart

### PR DESCRIPTION
**Pull request by tsvietOK**
Little bit increased acrylic opacity
Some code cleaning
For idea thanks to [Microsoft](https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/Helper/ThemeHelper.cs)
Known bugs: 
Sometimes the theme of datagrid is not updated
May be some freeze while changing theme

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Can you keep the acrylic opacity the same as it was before?

**Comment by tsvietOK** [comment]
@yair100 Let's compare:
Before:
Dark_Preview
Light_Preview
After:
Dark_Dev
Light_Dev

**Comment by yaichenbaum** [comment]
@tsvietOK I tested it myself, the first one is cleaner.

**Comment by tsvietOK** [comment]
@yair100 As you wish

**Comment by yaichenbaum** [comment]
If people really want we can add in a way to adjust it but I think that is taking it too far.

File: Files/MultilingualResources/Files.es-ES.xlf [hunk 1]
```diff
@@ -612,9 +612,9 @@
         </trans-unit>
         <trans-unit id="SettingsPreferencesTerminalApplicationsSampleProfiles.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>View sample profiles</source>
-          <target state="translated">Ver perfiles personalizados</target>
+          <target state="new">View sample profiles</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.es-ES.xlf:615]: Why was this reverted?

File: Files/Strings/es-ES/Resources.resw [hunk 2]
```diff
@@ -465,7 +465,4 @@
   <data name="SettingsExperimentalDoubleTapToRenameFiles.Text" xml:space="preserve">
     <value>Doble click para renombrar archivos</value>
   </data>
-  <data name="SettingsPreferencesTerminalApplicationsSampleProfiles.ToolTipService.ToolTip" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/Strings/es-ES/Resources.resw:468]: This shouldn't be removed.

File: Files/MultilingualResources/Files.es-ES.xlf (hunk 1 above)
Comment (tsvietOK) [reply Files/MultilingualResources/Files.es-ES.xlf:615]: Hmmm, may be some conflicts

Comment (yaichenbaum) [reply Files/MultilingualResources/Files.es-ES.xlf]: @tsvietOK Except for this, everything else looks good to go...

Comment (tsvietOK) [reply Files/MultilingualResources/Files.es-ES.xlf]: @yair100 I fixed this in 688c03937aa2825ca286a9ab81c5f44b062d5dec

**Review by yair100 — APPROVED** [review]

**Comment by tsvietOK** [comment]
@yair100 For some reason ModernNavigationToolbar exists at home page, do you have the same problem?
Edit: It is not part of this PR

**Comment by yaichenbaum** [comment]
@tsvietOK What is the issue you are seeing?

**Comment by tsvietOK** [comment]
@yair100 As you can see an screenshots above, bottom tool bar present at home page

**Comment by yaichenbaum** [comment]
@tsvietOK I do not see that, but I will test again.

**Comment by yaichenbaum** [comment]
@tsvietOK I don't see that issue.

**Comment by tsvietOK** [comment]
@yair100 I have found it. This caused by 8f3083ddaa4788173f7e000346bb5b333d3aa5d9
And this bug is language dependent. If i change var tabLocationHeader back to "New tab", toolbar invisible at home page.

**Comment by tsvietOK** [comment]
As i understand, home page detection based on "New tab" string

**Comment by yaichenbaum** [comment]
@tsvietOK To fix that we just need to update the check when navigating to a page to use the resource string.

**Comment by tsvietOK** [comment]
@yair100 Your changes fixes this bug.

**Pull request merged**